node/src/tasks/bsp_submit_proof.rs

```diff
@@ -261,10 +271,10 @@ where
     }
 }
 
-impl<FL, FS> BspSubmitProofTask<FL, FS>
+impl<FL, FSH> BspSubmitProofTask<FL, FSH>
 where
     FL: FileStorage<StorageProofsMerkleTrieLayout> + Send + Sync,
-    FS: ForestStorage<StorageProofsMerkleTrieLayout> + Send + Sync + 'static,
+    FSH: ForestStorageHandler<Key = NoKey> + Clone + Send + Sync + 'static,
```

Comment:
This trait could be "wrapped" as a `trait BspForestStorageHandler: ForestStorageHandler<Key = NoKey> + Clone + Send + Sync + 'static` so it's more clear what this is about.